iati/tests/management_unit_tests.py

```diff
@@ -0,0 +1,21 @@
+"""A module of unit tests for management commands."""
+import responses
+import pytest
+import json
+from home.management.commands.updatestatistics import ACTIVITY_URL, ORGANISATION_URL, get_total_num_activities, get_total_num_publishers
+from base_functional_tests import TEST_DATA_DIR
+
+
+@pytest.mark.django_db
+@responses.activate
+def test_update_statistics():
```

Comment:
This is great! Would suggest a test for the exceptions that we're hoping get raised too :-)